### PR TITLE
Split and tidy up stylesheets

### PR DESCRIPTION
**Pull request by bricesanchez**
* create components folder
* create global folder
* create mixins folder
* create plugins folder
* create sections folder
* Split and tidy up stylesheets

This PR is the first of a PR series in order to improve the modularity of the backend stylesheets.

This PR made **no UI changes**.

The goals are : 
* to give a cleaner way to manage stylesheets.
* to be able to improve current backend style.
* to give the ability to load few stylesheets in a new UI in order to be backward compatible.

Mergers, please test this PR before merging it.

### CODE REVIEW

File: core/app/assets/stylesheets/refinery/global/_colours.scss [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+$icon_preview_colour: $icon_default_colour;
+$icon_edit_colour: $icon_default_colour;
+$icon_add_colour: $icon_default_colour;
 
 $action_background_colour: lighten($icon_locale_colour,10%);
 $action_border_colour: $icon_locale_colour;
+
+// $icon_preview_colour: #7f00ff;   // violet
+// $icon_edit_colour:    #007fff;   // azure
+// $icon_add_colour:     #7fff00;   // chartreuse
```
Comment (parndt) [inline core/app/assets/stylesheets/refinery/global/_colours.scss]: did you mean to leave in the commented out rules?

Comment (bricesanchez) [inline core/app/assets/stylesheets/refinery/global/_colours.scss]: I could do a complete refactor of the current CSS for this PR but i think it's better to do this in an second PR in order to keep easy the review.

Comment (parndt) [inline core/app/assets/stylesheets/refinery/global/_colours.scss]: Oh, I see, these were already there.

**Comment by parndt** [comment]
Looks good to me

**Comment by anitagraham** [comment]
++

**Pull request merged**